### DISCUSSION TITLE
How to add the exported .html file inside a website

### DISCUSSION THREAD

**Discussion by imflash217** [marp-team/marp#269] [Q&A] [answered]
Hi,
Thanks for the great tool.
How do I embed the exported `.html` file inside my blog website's `div` and still show it as slides.
Thanks

**Comment by yhatt** [comment, marked as answer]
Try to use `<iframe>`.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

If you want to avoid using `<iframe>`, use [Marp Core](https://github.com/marp-team/marp-core) directly to generate raw HTML and CSS snippets. Marp CLI cannot output snippets because it is designed for outputting standalone HTML for common presentation use.

### Related

- #173

**Reply by luceat-lux-vestra** [reply]
Not possible to add an option to marp-cli?

**Reply by yhatt** [reply]
There is no plan because Marp CLI is designed just for generating standalone slide deck.
